src/languages/en.js

```diff
@@ -880,6 +880,7 @@ export default {
             disconnectYour: 'Disconnect your ',
             bankAccountAnyTransactions: ' bank account. Any outstanding transactions for this account will still complete.',
             clearProgress: 'Starting over will clear the progress you have made so far.',
+            deleteConfirmation: 'Are you sure?'
```

Comment:
This is a very common phrase. So We should move this under common.

Reply:
should I keep the key same? or should I change it to `areYouSure` or just `confirmation`?

Reply:
I think we should leave it. Currently, they're organized by the page they're used on, not by likeness of the phrase.

Reply:
So should I just fix the lint error?

Reply:
Yeah, do the lint fix.